src/test/scala/examples/Launcher.scala

```diff
@@ -27,7 +27,7 @@ object Launcher {
         }
       },
       "Life" -> { (backendName: String) =>
-        Driver(() => new Life(3), backendName) {
+        Driver(() => new Life(9), backendName) {
```

Comment:
Does it actually go up that much? The max address from a cursory glance is 27 or so.